courses/machine_learning/deepdive2/structured/solutions/4_bqml_linear_benchmark_babyweight.ipynb

```diff
@@ -0,0 +1,629 @@
+{
```

Comment:
<p>SELECT * and FROM &lt;table&gt; can be one liners</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I wanted to be explicit on what columns I am using from the table. I could have many more columns that I don't want in general. Plus it reinforces what our label and features are.</p>

Reply:
Sorry, not really following here. Isn't SELECT * the opposite of being explicit on what columns are being used?

Reply:
I think I see the confusion. I'm say lines 2/3 and 13/14 can be combined so there are two lines instead of 4.

Reply:
<p>Oh sorry, I thought you meant the select within the EVALUATE call and wanted me to change it to SELECT * lol. I kept the top select multiline because that is what the built-in BigQuery UI formatter changes it to and follows the standard indentation pattern with subqueries.</p>

Reply:
Oh nice, look at that. Thanks, I learned something new. It caught me off gaurd because all the sql style guides I've seen have SELECT * as one line, but let's default to the BQ format.